components/script/dom/element.rs

```diff
@@ -2174,6 +2175,32 @@ impl Element {
         };
     }
 
+    /// <https://www.w3.org/TR/CSP/#is-element-nonceable>
+    pub(crate) fn is_element_nonceable(&self, nonce_attr: DomRoot<Attr>) -> Option<String> {
```

Comment:
nit: since this is a function on `Element` I would call it `is_nonceable` instead.